tests/test_packages/test_skills_integration/test_simple_oracle.py

```diff
@@ -86,6 +86,13 @@ def test_oracle(self, erc20_contract, oracle_contract):
         )
         self.set_config(setting_path, oracle_address)
 
+        diff = self.difference_to_fetched_agent(
+            "fetchai/coin_price_oracle:0.3.0", oracle_agent_name
+        )
+        assert (
+            diff == []
+        ), "Difference between created and fetched project for files={}".format(diff)
```

Comment:
What about checking for the other agent?

Reply:
Good catch, thanks. This agent was not yet added to `packages/fetchai/agents`, so this is a good reminder to do this.